tests/runtest.py

```diff
@@ -32,6 +32,8 @@ class TestBase:
                       '-fno-omit-frame-pointer', '-D_FORTIFY_SOURCE=0']
 
     def __init__(self, name, result, lang='C', cflags='', ldflags='', sort='task'):
+        # assume that this script is executed in uftrace/tests directory.
+        self.srcdir = os.path.dirname(os.getcwd())
```

Comment:
The assumption is correct otherwise it cannot find any test case.

Reply:
Would it be better to remove the comment?

Reply:
I don't care :)